### ISSUE TITLE
analyze timings

### ISSUE THREAD

**Issue by eclare108213** [CICE-Consortium/CICE#263]
Update the timings plot for v6.  

https://github.com/CICE-Consortium/CICE/blob/2f79cb314051ece262b7497dd197831559d66ee7/doc/source/user_guide/ug_implementation.rst

Figure :ref:`fig-timings`

**Comment by eclare108213** [comment]
Do any of the modeling centers running CICE output timings in your regular runs?  If so, please paste them here.

**Comment by DeniseWorthen** [comment]
@eclare108213 Are you referring to the information in the ice_diag.d under "Timing information"? We do have that information for all our regular tests (1deg, 1/2deg and 1/4 deg tripole) but we keep no statistics, so the numbers would be a single case.

**Comment by eclare108213** [comment]
Yes, that's what I'm after.  If you have a bunch of them and they are easy to collate, could you put them in a document or spreadsheet, and post it here?  Otherwise one example per grid would be a great place to start.  Thanks!

**Comment by DeniseWorthen** [comment]
Here's a set from a recent run on platform Hera. They're for different run lengths. The 1deg is 24 hrs, the 1/2deg is 30hrs and the 1/4deg is only 6 hours. 

It isn't easy to put them into a spreadsheet because of they way they're written. Having the min,mean,max on one line would be a lot easier.

[hera.timings.1deg.txt](https://github.com/CICE-Consortium/CICE/files/11223864/hera.timings.1deg.txt)
[hera.timings.p5deg.txt](https://github.com/CICE-Consortium/CICE/files/11223865/hera.timings.p5deg.txt)
[hera.timings.p25deg.txt](https://github.com/CICE-Consortium/CICE/files/11223866/hera.timings.p25deg.txt)

**Comment by eclare108213** [comment]
Thanks @DeniseWorthen, this is very helpful.  Please, would you post one of the full ice_diag.d files?  I'll grab some configuration specifics (like ncat, nilyr, etc) from it.

**Comment by DeniseWorthen** [comment]
Here is the 1/4deg ice_diag file. All the configurations use the same basic configuration.

[ice_diag.d.p25deg.txt](https://github.com/CICE-Consortium/CICE/files/11226026/ice_diag.d.p25deg.txt)
